### PR TITLE
Update RViz for 2024 project

### PR DESCRIPTION
**Pull request by steph-lheureux**
# Summary
This PR requests the merge of the updated RViz setup for the 2024 robot. #197 

# Additions
- Updated `.stl` and `.udrf` for the SeaHawk II
- Created [`rviz.md`](https://github.com/CabrilloRoboticsClub/cabrillo_rov_2023/blob/rviz/doc/rviz.md) which describes the usage and updating of the current RViz
- Updated launch file to unset `GTK_PATH` automatically if using VS code installed with snap

### CODE REVIEW

**Comment by OrionOth** [comment]
Before anyone reviews this, consider condensing your commit history?

File: doc/rviz.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# RViz
+
+## Introduction
+[Rviz](http://wiki.ros.org/rviz) (ROS Visualization) is a 3D tool which enables developers to view the functionality of a robot virtually. This is useful for software development before the physical robot is completed or for remote work. Our project uses RViz to visualize the thrust vectors of the motors.
```
Comment (liamgilligan) [inline doc/rviz.md]: It visualizes the water flow, not the thrust vectors

File: src/seahawk/seahawk_deck/input_xbox_one.py
Comment (liamgilligan) [inline src/seahawk/seahawk_deck/input_xbox_one.py:1]: Delete this

File: src/seahawk_description/launch/display.launch.py [hunk 2]
```diff
@@ -54,7 +55,7 @@ def generate_launch_description():
         condition=UnlessCondition(LaunchConfiguration('enable_joint_state_pub_gui'))
     )
 
-    # TODO: Is this node useful for anything? They include it in the exported
+    # TODO: Is this node useful for anything? They inclu de it in the exported
```
Comment (liamgilligan) [inline src/seahawk_description/launch/display.launch.py]: Space

**Review by liamgilligan — APPROVED** [review]
Looks good other than the comments I left. Good work!

**Pull request merged**